styleguide/source/_patterns/03-organisms/resource-tabs.twig

```diff
@@ -0,0 +1,247 @@
+<div class="ama__resource-tabs">
+  <ul class="ama__resource-tabs__nav">
```

Comment:
This should go through loop. Such as:
{% for resource-tab in ama__resource-tabs %}
  {% include "molecule/resource-tab.twig" %}
{% endfor %}

Reply:
or better yet. Can we use the list atom here instead of creating a new molecule?